devops/Makefile

```diff
@@ -630,4 +676,9 @@ db-backup-deploy-delete-mongodb:
   	networkpolicy/$(BACKUP_MONGODB_APP_NAME)
 # The below PVCs are not deleted.
 # pvc/$(BACKUP_MONGODB_APP_NAME)-pvc
-# pvc/$(BACKUP_MONGODB_APP_NAME)-verification-pvc
\ No newline at end of file
+# pvc/$(BACKUP_MONGODB_APP_NAME)-verification-pvc
```

Comment:
for my info, what is `pvc`

Reply:
It is the Openshift permanent storage: https://docs.openshift.com/container-platform/4.11/storage/understanding-persistent-storage.html
It is used when we need to store data in a way that the PODs can come up and down or even be removed but the data must be there, for instance, for databases like Postgres and Redis.
For PODs like Web/API we do not care about where the files are gonna be because everything can be destroyed and recreated, for DB we need the data to be persistent.